src/index.js

```diff
@@ -167,9 +167,15 @@ function generateFallbackFormatter(isoCode, locale, numDecimals = 2) {
   } else {
     return {
       format: (value) => {
+        let formattedValue = value
+        // try using Intl.NumberFormat when possible to support max significant digits 
+        try {
+          formattedValue = new Intl.NumberFormat('en', { maximumSignificantDigits }).format(value)          
```

Comment:
@adrian-gecko `locale` is available in the scope of the function

```suggestion
          formattedValue = new Intl.NumberFormat(locale, { maximumSignificantDigits }).format(value)          
```